app/templates/grunt/config/util/injector.js

```diff
@@ -111,6 +111,7 @@ var taskConfig = function(grunt) {
             filePath = filePath.replace('/client/styles/', '');
           }<% } else { %>
           filePath = filePath.replace('/client/styles/', '');<% } %>
+          filePath = filePath.replace(/_/, '');
```

Comment:
This code might cause issues as it will remove the first underscore from any path.
ex: `myfolder/my_folder/_mypartial` --> `myfolder/myfolder/_mypartial`
This is not the desired effect.

I think a better way to implement what you are trying to do is a bit more complicated, but here is an example that should work:

```js
// Assume filePath is 'myfolder/my_folder/_mypartial';

// Split filepath into an array with '/' as the delimiter
var filePathArray = filePath.split('/');

// Remove filename from filePathArray and assign it to fileName
var fileName = filePathArray.pop();

// Check to see if the first character of fileName is '_'
if (fileName.split('')[0]) {
  // if true, remove the '_'
  fileName = fileName.replace('_', '');
}

// Add filename back to filePathArray
filePathArray.push(fileName);

// Join array into back into filepath string
filePath = filePathArray.join('/');
```

Try that out and see if you get the desired effect (removing the filename underscore from scss/sass files). Feel free to improve this as well, as there is probably a more elegant solution.